packages/ui/src/icons/console-icon.native.tsx

```diff
@@ -1,6 +1,7 @@
 import Console from '../assets/icons/console.svg';
 import { Icon, IconProps } from './icon/icon.native';
 
+// Is this icon outdated?
```

Comment:
Lets double check this with design

Reply:
Thx, I'm communicating with design team on these and might need to circle back to them separately, but I'll try to address them here before merging.